### PR TITLE
Change to the new hostname of LIMS-production

### PR DESCRIPTION
**Pull request by chuan-wang**
Note: do not deploy on ngi-internal until the new production VM officially replace the old LIMS.

### CODE REVIEW

**Comment by aanil** [comment]
We should probably change these to read from the config rather than hardcoding the links in the code.

File: status/projects.py [hunk 1]
```diff
@@ -575,7 +575,7 @@ def get(self, project, sample, step):
     @staticmethod
     def get_caliper_image(url):
         """returns a base64 string of the caliper image asked"""
-        #url pattern: sftp://genologics.scilifelab.se/home/glsftp/clarity/year/month/processid/artifact-id-file-id.png
+        #url pattern: sftp://ngi-lims-prod.scilifelab.se/home/glsftp/clarity/year/month/processid/artifact-id-file-id.png
```
Comment (aanil) [inline status/projects.py:578]: So the server name that we ssh to will be renamed too?

**Review by aanil — COMMENTED** [review]
Looks good! Is it available on stage?

File: status/projects.py (hunk 1 above)
Comment (chuan-wang) [reply status/projects.py:578]: Yes. genologics.scilifelab.se will be dismissed from service.

**Comment by chuan-wang** [comment]
> Looks good! Is it available on stage?

No, I haven't deployed anywhere. Just tested it locally.

**Review by aanil — APPROVED** [review]
Seems to work on stage!

**Pull request merged**